cmd/minikube/cmd/start.go

```diff
@@ -757,10 +757,41 @@ func memoryLimits(drvName string) (int, int, error) {
 			return -1, -1, err
 		}
 		containerLimit = int(s.TotalMemory / 1024 / 1024)
+		myabeAdviceDockerResources(containerLimit, sysLimit, s.CPUs, drvName)
 	}
+
 	return sysLimit, containerLimit, nil
 }
 
+func myabeAdviceDockerResources(containerLimit int, sysLimit int, cpus int, drvName string) {
+	if drvName == oci.Docker && runtime.GOOS != "linux" {
+		if containerLimit < 1991 {
+			out.T(out.Conflict, `Your Docker Desktop has only {{.container_limit}} memory. Increase memory to at least 2.5 GB or more:
```

Comment:
This should probably go into validateMemorySize and reuse minUsableMem so that there is standardization between Docker and non-Docker recommendations.



Should this be out.WarningT? It should definitely go to stderr, anyways.